docs/source/kedro_project_setup/configuration.md

```diff
@@ -12,6 +12,14 @@ You can also specify a source directory for the configuration files at run time
 ```bash
 kedro run --conf-source=<path-to-new-conf-directory>
 ```
+
+If you're using the [`OmegaConfigLoader`](/kedro.config.OmegaConfigLoader) you can also read configuration from a compressed file in `zip` or `tar` format. The two commands below are examples of how to reference a `tar.gz` or `zip` file.
```

Comment:
```suggestion
If you're using the [`OmegaConfigLoader`](/kedro.config.OmegaConfigLoader) you can also read configuration from a compressed file in `zip` or `tar` format. The two commands below are examples of how to reference a `tar.gz` or `zip` file:

```

Reply:
Adding an extra blank line between the text and the code snippet to ensure it renders.